src/Components/Pages/Home.js

```diff
@@ -1,7 +1,11 @@
+// import TradingChart from './TradingChart';
```

Comment:
Need to pull in changes from Main branch

Reply:
Never mind. I fixed the conflict for Home.js. If you fix the package.json conflicts, you can merge.